docs/en/tutorials/getting-started.rst

```diff
@@ -274,7 +274,7 @@ but you only need to choose one.
         <?php
         // src/Product.php
 
-        use Doctrine\ORM\Annotation as ORM;
+        use Doctrine\ORM\Mapping as ORM;
```

Comment:
This is because `master` has a completely different structure. Please refer to the `2.6` branch for the current docs :-)